src/components/login-component/LoginComponent.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+function LoginComponent() {
+  return (
+    <div>
+      <section className="login-section">
+        <h2 className="align-center">Log In</h2>
+        <form className="login-form">
+          <div className="input login-input">
+            <input type="text" placeholder="Enter Username" />
+          </div>
+          <div className="input login-input-password">
+            <input type="password" placeholder="Enter Password" />
+            <i className="fa-solid fa-eye-slash" id="toggle-password-login"></i>
+          </div>
+          <div className="button button-icon login-button">
+            {" "}
+            <span> Log In </span>{" "}
+          </div>
+          <p className="login-signup">
+            Not A Member Yet?<a href="./signup.html"> Signup</a> instead.
```

Comment:
you can use `Link` from react router instead of `a` tag

Reply:
Fixed